docs/articles/documentation/guides/advanced-guides/intercept-http-requests.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: docs
+title: Intercept HTTP Requests
+permalink: /documentation/guides/advanced-guides/intercept-http-requests.html
+---
+# Intercept HTTP Requests
+
+This section describes how to handle HTTP requests in your tests. TestCafe allows you to log them and mock the responses out of the box. You can also create a custom HTTP request hook, which allows you, for instance, to emulate authentications like **Kerberos** or **Client Certificate Authentication**.
```

Comment:
> log them

Log whom? :) 

Also, are the responses coming out of the box, or is it an out-of-the-box feature?